src/Doctrine/ORM/ProgramRepository.php

```diff
@@ -17,7 +18,7 @@ public function findOneByChannel(ChannelInterface $channel): ?ProgramInterface
         $obj = $this->createQueryBuilder('o')
             ->andWhere('o.channel = :channel')
             ->andWhere('o.enabled = true')
-            ->setParameter('channel', $channel)
+            ->setParameter('channel', $channel, Types::OBJECT)
```

Comment:
Usually I suppress this Psalm error that demands this because I am not sure what to actually input in that third parameter always. Are you sure about that? :)